File: reni-backend/src/main/java/pe/gob/mimp/reni/dto/request/ReporteIntervencionListarRequest.java
package pe.gob.mimp.reni.dto.request;

import java.io.Serializable;
import java.util.List;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class ReporteIntervencionListarRequest implements Serializable {
	private static final long serialVersionUID = 1L;

	private String sqlFields;
	private String sqlTables;
	private List<ReporteIntervencionListarDetRequest> listaMapeo;
	private String listaEstructura;
}

File: reni-frontend/src/main/reni-frontend/src/app/modules/intranet/dto/request/reporte-intervencion-listar.request.ts
import { ReporteIntervencionListarDetRequest } from "./reporte-intervencion-listar-det.request";

export class ReporteIntervencionListarRequest {
    sqlFields: string;
    sqlTables: string;
    listaMapeo: ReporteIntervencionListarDetRequest[];
    listaEstructura: string;
}